### PR TITLE
Make schema HTML support events that are tuples at root level

### PR DESCRIPTION
**Pull request by yli-cpr**

### CODE REVIEW

File: templates/html/.partials/schema.html [hunk 1]
```diff
@@ -4,6 +4,25 @@
   <h4>{{schemaName}}</h4>
   {{/unless}}
 
+  {{#if schema.items.[0]}}
```
Comment (fmvilas) [inline templates/html/.partials/schema.html]: Just out of curiosity: doesn't `schema.items.length` work?

Comment (yli-cpr) [reply templates/html/.partials/schema.html]: Cool. Just verified that it works.

Comment (fmvilas) [reply templates/html/.partials/schema.html]: Awesome! I wasn't sure. Handlebars drives me nuts sometimes.

**Comment by fmvilas** [comment]
It's now [deployed to npm](https://www.npmjs.com/package/asyncapi-generator). Thanks for the PR, @yli-cpr!

**Comment by yli-cpr** [comment]
Thank you @fmvilas !

**Pull request merged**